aviary/subsystems/aerodynamics/gasp_based/gaspaero.py

```diff
@@ -1389,10 +1391,8 @@ def setup(self):
                 # so ensure this is what's passed to DragCoef
                 promotes_outputs=[("CL", "CL_full_flaps")],
             )
-            # Note: alpha is not an output
+            warnings.warn("Alpha is NOT an output from LowSpeedAero.")
```

Comment:
@johnjasa and @Kenneth-T-Moore
could we add a balance comp here (or in CLFromLift) so that this does actually output alpha?
![image](https://github.com/OpenMDAO/Aviary/assets/51181861/28326021-b8f8-4da0-9478-73bfcfc22118)

Reply:
@crecine : I am not familiar enough with Gasp aero, but I think that the issue is that the component that computes alpha from the lift_curve_slope, but we only do this for the clean aero, and this is low speed aero.  I'm not sure if that calculation is valid in low speed.

What you suggest about the balance comp would require a solver, and it would have to actually do something if we varied alpha (and we would have to have the right independents to watch.)  Probably way beyond this story.

Reply:
Ahh, that makes sense. Thanks